app/helpers/ebird_helper.rb

```diff
@@ -69,17 +66,9 @@ def hav_distance(geo_a, geo_b, miles=false)
   end
 
   def getImageSrc(bird_data)
-    img_src = 'never set'
-    begin
-      img_src = getImageFromName(bird_data["comName"])
-    rescue NoMethodError => e1
-      puts e1
-      begin
-        img_src = getImageFromName(bird_data["sciName"])
-      rescue NoMethodError => e2
-        puts e2
-        img_src = nil
-      end
+    img_src = nil
+    if bird_data
+      img_src = getImageFromName(bird_data["comName"] || bird_data["sciName"])
```

Comment:
all ebird birds have a com name and a sci name, just some Wikipedia article dont exist under them, getImageFromName throws an exception when that happens